eureka2-metrics-spectator/src/main/java/com/netflix/eureka2/metric/server/SpectatorReplicationChannelMetrics.java

```diff
@@ -14,17 +14,18 @@
  * limitations under the License.
  */
 
-package com.netflix.eureka2.server.metric;
+package com.netflix.eureka2.metric.server;
 
+import com.netflix.eureka2.channel.ReplicationChannel.STATE;
 import com.netflix.eureka2.metric.AbstractStateMachineMetrics;
-import com.netflix.eureka2.server.channel.ReceiverReplicationChannel.STATES;
+import com.netflix.spectator.api.ExtendedRegistry;
 
 /**
  * @author Tomasz Bak
  */
-public class ReplicationChannelMetrics extends AbstractStateMachineMetrics<STATES> {
+public class SpectatorReplicationChannelMetrics extends AbstractStateMachineMetrics<STATE> implements ReplicationChannelMetrics {
 
-    public ReplicationChannelMetrics() {
-        super("server", STATES.class);
+    public SpectatorReplicationChannelMetrics(ExtendedRegistry registry, String server) {
+        super(registry, "server", STATE.class);
```

Comment:
I need to cleanup metric names. Probably centralizing all these names in one class would be best.